components/sandbox/index.js

```diff
@@ -134,7 +134,6 @@ export default class Sandbox extends Component {
 				document.body.style.position = 'absolute';
 				document.body.style.width = '100%';
 				document.body.setAttribute( 'data-resizable-iframe-connected', '' );
-
```

Comment:
Accidental? :)

Reply:
`git commit -a --amend`

YOU SAW NOTHING ;)